test/basic.test.js

```diff
@@ -516,6 +516,11 @@ test('auto pino.destination with a string as second argument', async ({ same })
   })
 })
 
+test('creates pretty write stream', async ({ is }) => {
+  const prettyStream = pino.createPrettyWriteStream()
+  is(typeof prettyStream.write, 'function')
+})
+
```

Comment:
Can you add a couple more tests?

1. with a custom prettifier
2. that calls pino-pretty

There should also be a test file specific for pretty-print, so maybe they can live there.

Reply:
Yes,... will do so. Currently working on the docs already.